admin_manual/maintenance/migrating.rst

```diff
@@ -29,3 +29,112 @@ To start, let us be specific about the use case. A configured ownCloud instance
 
 #.  Change the ``CNAME`` entry in the DNS to point your users to the new
     location.
+
+
+
+=================
+Example migration
+=================
+
+.. note:: For this example to work, you need **ssh** on both servers and the **PermitRootLogin** to be set to "yes" on the new server.
+
+Install SSH::
+
+   apt install openssh-server openssh-client -y
+
+Edit SSH-Config::
+
+   nano /etc/ssh/sshd_config
+
+Change PermitRootLogin to yes::
+
+   PermitRootLogin yes
+
+=========
+Migration
+=========
+
+1. Install ownCloud on new server
+
+2. Put original server in maintenance mode:
+
+Go in owncloud dir::
+
+      cd /var/www/owncloud/
+
+
+Switch to maintenance mode::
+
+      sudo -u www-data php occ maintenance:mode --on
+
+
+wait for 6-7 min and stop apache2::
+
+   service apache2 stop
+
+3. Export and Import the database
+
+Go in owncloud dir::
+
+      cd /var/www/owncloud/
+
+Export on original server
+
+SYNOPSIS::
+
+   mysqldump --lock-tables -h [server] -u [username] -p[password] [db_name] > owncloud-dbbackup_`date +"%Y%m%d"`.bak
+
+Example::
+
+   mysqldump --lock-tables -h localhost -u admin -ppassword owncloud > owncloud-dbbackup.bak
+
+.. note:: Parameters are set when creating database for ownCloud
+
+::
+
+   CREATE DATABASE IF NOT EXISTS owncloud;
```

Comment:
Yes, it's done on the original server. I wrote this command to remind the user where he set the parameters which are needed for the mysqldump command, in case he forgot it.